DAL/SQLHelper.cs

```diff
@@ -533,17 +533,18 @@ public void UpdateContentItem(StructuredContent_ContentType content_type, dynami
                     if (content_field.is_system == false)
                     {
                         sbUpdateContentItem.Append(", [" + content_field.column_name + "]");
-
                         sbUpdateContentItem.Append("=");
 
-                        object value = content_item[content_field.column_name];
-
-                        if (value == null)
+                        var field_value = content_item[content_field.column_name];
```

Comment:
If `value` here is renamed to `field_value`, then it needs to be renamed everywhere is gets referenced...

```
 var field_value = content_item[content_field.column_name];
                        
                        if (field_value == default(dynamic))
                        {
                            sbUpdateContentItem.Append("null");
                        }
                        else
                        {
                            //object value = content_item[content_field.column_name];

                            switch (content_field.data_type)
                            {
                                case (int)Enums.DataTypes.bit:
                                    if (bool.Parse(value.ToString()))
```

There is in Visual Studio a shortcut to do it everywhere (CTRL+R+R)